packages/web/components/PageHeader.tsx

```diff
@@ -77,11 +79,42 @@ const SubMenuItem: React.FC<React.ComponentProps<typeof MetaLink>> = ({
         fontWeight="normal"
         color="whiteAlpha.700"
         margin={3}
-        className="filter-effect"
-        // backgroundImage={`url(${MetaBoxButton})`}
+        boxShadow="0 0 2rem rgba(0, 0, 0, 0)"
+        _before={{
+          content: "''",
+          position: 'absolute',
+          display: 'none',
+          top: 0,
+          left: 0,
+          width: '100%',
+          height: '100%',
+          transform: 'translate3d(-120%, 0, 0)',
+          filter: 'blur(15px)',
+          background:
+            'linear-gradient(45deg, transparent, rgba(255, 255, 255, 0), rgba(255, 255, 255, 0.06), rgba(255, 255, 255, 0), transparent)',
+          transition: 'all 0.4s 0.3s ease-in-out',
+          zIndex: 1,
+        }}
+        _after={{
+          content: "''",
+          position: 'absolute',
+          display: 'block',
+          top: 0,
+          left: 0,
+          width: '100%',
+          height: '100%',
+          filter: 'blur(10px)',
+          transform: 'translate3d(-120%, 0, 0)',
+          background:
+            'linear-gradient(45deg, transparent, rgba(255, 255, 255, 0), rgba(255, 255, 255, 0.06), rgba(255, 255, 255, 0), transparent)',
```

Comment:
A lot of repeated code / styles, should consolidate and clean it up. You dont need 90% of these if you follow Chakra UI method of styling

Reply:
The bulk of the repetition was a `_before` with a `display` of `none`. Removing it didn't affect the display, so I removed it.